app/controllers/providers/link_case/confirmations_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Providers
+  module LinkCase
+    class ConfirmationsController < ProviderBaseController
+      prefix_step_with :link_case
+
+      def show
+        @linked_application_types = LinkedApplicationType.all
+
+        if legal_aid_application.copy_case?
+          destroy_linked_application
```

Comment:
what scenario does this cover, do we need this?

Reply:
to answer my own question i think this is to cover the flow of a copied case being direct to this page, skipping the invitation and search pages. In some looping scenarios this could result in an existing linked application which we would need to delete here.